plugins/database/mysql/mysql-database-plugin/main.go

```diff
@@ -5,7 +5,7 @@ import (
 	"os"
 
 	"github.com/hashicorp/vault/plugins/database/mysql"
-	dbplugin "github.com/hashicorp/vault/sdk/database/dbplugin/v5"
+	"github.com/hashicorp/vault/sdk/database/dbplugin/v5"
```

Comment:
Looks like we missed the `ServeMultiplex` operation on the MySQL DB engine

Reply:
Yeah you're right, it's a bit misleading but I did the MySQL project while adding the opt-out flag as this was the system I used to test the feature:
https://github.com/hashicorp/vault/pull/16972/files#diff-aaf53a599c386295cbe7f2ba70be6791e775d5cd51ca8b352e5887f7d8d51b74R24

I can add the same modifications here so the PR is "unified" if someone looks back into the history at some point.

Reply:
Ah I see okay I missed that, sorry; I think whatever you prefer is fine. If you'd prefer to keep the MySQL updates in the opt-out PR, I'd suggest adding a link/reference to that code stating that the MySQL updates have been made there. This PR's description might be good spot for that link :)